### PR TITLE
Add Google ad conversion metrics to docs.sourcegraph.com

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

File: doc/_resources/assets/docs.js [hunk 1]
```diff
@@ -169,4 +173,19 @@ window.sgdocs = (() => {
...
+    if (window && window.gtag) {
+      window.gtag('event', 'conversion', {
+        'send_to': 'AW-868484203/vOYoCOCUj7EBEOuIkJ4D',
+      });
+    }
+  }
+
+  function startSourcegraphCommandsInit() {
+    START_SOURCEGRAPH_COMMAND_SNIPPETS.forEach(el => {
+      el.onclick = gaConversionOnStartSourcegraphCommands
```
Comment (ryan-blunden) [inline doc/_resources/assets/docs.js]: Change to use `addEventListener` API

Comment (dadlerj) [reply doc/_resources/assets/docs.js]: So my fear with addEventListener was that I don't know how Gatsby component lifecycles work. If I just "add" the event listener here, how often is this init code run? On initial pageload, or on every page change? When should I _remove_ the event listener? What if the user clicks "back" in their browser? It'd be very bad to accidentally re-attach this event handler multiple times.

LMK if you have guidance on this.

Comment (ryan-blunden) [reply doc/_resources/assets/docs.js]: As docsite is not a single page app, the init code is executed once (on dom ready), so there is no need to remove the event listener.

Comment (dadlerj) [reply doc/_resources/assets/docs.js]: Hah, I assumed it was! Thanks for confirming

**Comment by dadlerj** [comment]
Updated. Mind taking a look again @ryan-blunden ?

**Comment by dadlerj** [comment]
Bump @ryan-blunden

File: doc/_resources/assets/docs.js [hunk 2]
```diff
@@ -154,4 +158,19 @@ window.sgdocs = (() => {
...
+  function gaConversionOnStartSourcegraphCommands() {
+    if (window && window.gtag) {
+      window.gtag('event', 'conversion', {
+        'send_to': 'AW-868484203/vOYoCOCUj7EBEOuIkJ4D',
+      });
+    }
+  }
+
+  function startSourcegraphCommandsInit() {
+    START_SOURCEGRAPH_COMMAND_SNIPPETS.forEach(el => {
```
Comment (ryan-blunden) [inline doc/_resources/assets/docs.js]: Are you expecting multiple `docker run` command blocks to be present on a single page? If not, consider changing to `querySelector`, negating the need for the `forEach`.

Comment (dadlerj) [reply doc/_resources/assets/docs.js]: Hmm good point, there are several of these blocks across the site, but just one per page.

Comment (dadlerj) [reply doc/_resources/assets/docs.js]: Actually, this feels like over-optimizing... In the tiny chance that someone adds a second snippet on the quickstart page, I don't want to miss it!

**Pull request merged**